dnf.spec

```diff
@@ -299,15 +299,17 @@ popd
 %dir %{confdir}/aliases.d
 %exclude %{confdir}/aliases.d/zypper.conf
 %if %{without dnf5_obsoletes_dnf}
+# If DNF5 does not obsolete DNF ownership of dnf.conf should be DNF's
 %config(noreplace) %{confdir}/%{name}.conf
-%endif
-
+# If DNF5 does not obsolete DNF, protected.d/dnf.conf should be owned by DNF
+%config(noreplace) %{confdir}/protected.d/%{name}.conf
```

Comment:
Sorry, one more thing, I think we can move this line out of the `if` actually, and not have the `else` branch. We don't need the `%config` without `noreplace` any longer.